gsl/gsl_assert

```diff
@@ -62,10 +72,25 @@ struct fail_fast : public std::runtime_error
 
 #elif defined(GSL_TERMINATE_ON_CONTRACT_VIOLATION)
 
+#if defined(GSL_MESSAGE_ON_CONTRACT_VIOLATION)
+#define Expects(cond)                                                                              \
+    if (!(cond)) {                                                                                 \
+        std::cerr << "GSL: Precondition failure at "                                               \
+                     __FILE__ ": " GSL_STRINGIFY(__LINE__) "\n";                                   \
```

Comment:
Also please remove space after `__FILE__`, CLion can automatically detect format `FILE:LINE` in output

Reply:
? I'm not sure I see any space after `__FILE__` ..

Reply:
Sorry, I mean after colon `": "`